runners/core-java/src/main/java/org/apache/beam/runners/core/SimpleDoFnRunner.java

```diff
@@ -213,19 +200,19 @@ public void finishBundle() {
     }
   }
 
-  private DoFnStartBundleContext<InputT, OutputT> createStartBundleContext(
+  private DoFnStartBundleContext createStartBundleContext(
       DoFn<InputT, OutputT> fn, DoFnContext<InputT, OutputT> context) {
```

Comment:
Do we need any args here? can't we do it the same way `createProcessContext` does it, and use the fields directly?